worker-workflow/src/test/java/com/github/cafdataprocessing/workflow/WorkflowWorkerTest.java

```diff
@@ -169,5 +216,933 @@ public void subDocumentPassConditionTest() throws Exception {
                 actionExpectationsBuilder.build());
 
     }
+    
+    @Test
```

Comment:
Since these tests are against the workflow-control.js we could extract them to a new test class. WorkflowControlTest.java